frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -157,11 +157,19 @@ function ReaderDictionary:init()
     -- Prepare the -u options to give to sdcv the dictionary order and if some are disabled
     self:updateSdcvDictNamesOptions()
 
+    self:registerKeyEvents()
```

Comment:
Is the location based on anything in particular? If I'm not mistaken most existing inits put it at the top.